pkg/deploy/api/types.go

```diff
@@ -206,3 +206,13 @@ type DeploymentConfigList struct {
 	kapi.ListMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
 	Items         []DeploymentConfig `json:"items" yaml:"items"`
 }
+
+type RollbackSpec struct {
+	kapi.TypeMeta `json:",inline" yaml:",inline"`
```

Comment:
How does that play out from the client side? What JSON would the user be required to construct?